recipes/xorg-makedepend/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import re
+
+required_conan_version = ">=1.33.0"
+
+
+class XorgMakedepend(ConanFile):
+    name = "xorg-makedepend"
+    description = "Utility to parse C source files to make dependency lists for Makefiles"
+    topics = ("conan", "xorg", "dependency", "obsolete")
+    license = "MIT"
+    homepage = "https://gitlab.freedesktop.org/xorg/util/cf"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+
+    exports_sources = "patches/*"
+    generators = "pkg_config"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def requirements(self):
+        self.requires("xorg-macros/1.19.3")
+        self.requires("xorg-proto/2021.4")
+
+    def build_requirements(self):
+        self.build_requires("pkgconf/1.7.4")
+
+    def validate(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("Windows is not supported by xorg-gccmakedep")
```

Comment:
copy-paste devil :imp: 
```suggestion
            raise ConanInvalidConfiguration("Windows is not supported by xorg-makedepend")
```